### PR TITLE
cancelling pool incurs penalty

### PR DESCRIPTION
**Pull request by prateekreddy**

### CODE REVIEW

File: contracts/Pool/Pool.sol [hunk 1]
```diff
@@ -570,6 +610,10 @@ contract Pool is Initializable, IPool, ReentrancyGuard {
             _balance = _balance.mul(_totalAsset).div(poolToken.totalSupply());
         }
 
+        if(_loanStatus == LoanStatus.CANCELLED) {
+            _balance = _balance.mul(poolVars.penalityLiquidityAmount).div(poolToken.totalSupply());
```
Comment (ritik99) [inline contracts/Pool/Pool.sol:614]: Shouldn't this be _balance = _balance(1 + penalityLiquidityAmount / totalSupply)?

File: contracts/Pool/Pool.sol [hunk 2]
```diff
@@ -570,6 +610,10 @@ contract Pool is Initializable, IPool, ReentrancyGuard {
             _balance = _balance.mul(_totalAsset).div(poolToken.totalSupply());
```
Comment (ritik99) [inline contracts/Pool/Pool.sol:610]: Are we handling the case of withdrawing liquidity after lender margin called separately? Might need separate logic if lender withdrawing liquidity has margin called (and then the borrower is getting liquidated) vs when the borrower defaults by missing repayment

**Review by ritik99 — CHANGES_REQUESTED** [review]
Added two comments

File: contracts/Pool/Pool.sol (hunk 1 above)
Comment (prateekreddy) [reply contracts/Pool/Pool.sol:614]: correct, fixing it.

File: contracts/Pool/Pool.sol (hunk 2 above)
Comment (prateekreddy) [reply contracts/Pool/Pool.sol:610]: When lender margin calls, liquidator directly transfers the lent tokens to lender. So the balance of the lender will be 0.

**Comment by ritik99** [comment]
Looks good!

**Review by ritik99 — APPROVED** [review]

**Pull request merged**